src/main/resources/jenkins/metrics/api/MetricsAccessKey/resource.js

```diff
@@ -22,41 +22,41 @@
  * THE SOFTWARE.
  */
 function saveApiToken(button) {
-  if (button.hasClassName('request-pending')) {
+  if (button.classList.contains('request-pending')) {
     return;
   }
-  button.addClassName('request-pending');
-  const repeatedChunk = button.up('.repeated-chunk');
+  button.classList.add('request-pending');
+  const repeatedChunk = button.closest('.repeated-chunk');
 
   const targetUrl = button.getAttribute('data-target-url');
-  new Ajax.Request(targetUrl, {
+  fetch(targetUrl, {
     method: 'post',
```

Comment:
This is technically a change in behavior, as the Jenkins version of `Ajax.Request` was patched to automatically add a CSRF crumb, but the new `fetch` code in this PR does not do that. To make the new code equivalent to the old:

```suggestion
    method: 'post',
    headers: crumb.wrap({}),
```

Reply:
Thanks @basil. I included this in https://github.com/jenkinsci/metrics-plugin/pull/219/commits/5307c5dd3bac0b83437002b647f60b85542303aa